src/services/chapter.ts

```diff
@@ -199,8 +196,6 @@ class ChapterParser {
             const author = {
                 avatar: $(el).find('img').attr('src'),
                 name: $(el).find('span.name').find('a').text().split('@')[0],
-                premium: $(el).find('div.upcase.label.bg-yellow-crusta')
-                    .text().length !== 0,
```

Comment:
why remove this, is there no way to tell if a comment author has premium anymore?

Reply:
yeah